src/color/creating_reading.js

```diff
@@ -242,6 +237,18 @@ p5.prototype.brightness = function(c) {
  * </code>
  * </div>
  */
+
+/**
+ * @method color
+ * @param  {Number|String} v1      red or hue value relative to
+ *                                 the current color range, or a color string
+ * @param  {Number}        v2      green or saturation value
+ *                                 relative to the current color range
+ * @param  {Number}        v3      blue or brightness value
+ *                                 relative to the current color range
+ * @param  {Number}        [alpha]
```

Comment:
Because of the way the parameter merging logic works, we don't need to actually describe `[alpha]` because we've already documented it in the previous version of the method.